util/fipstools/delocate/delocate.go

```diff
@@ -488,6 +490,19 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 		}
 
 		return d.loadAarch64Address(statement, targetReg, symbol, offset)
+	case "bl":
+		// We were relying on symbols defined with ".comm" to populate bssAccessorsNeeded,
+		// but gcc release does not use ".comm" to define common symbols. The accessor
+		// functions are defined with a ".type $symbol %object" followed with a
```

Comment:
So `%object` was used for data objects and functions that needed `bss_get()`? (If I understand correctly, `DEFINE_METHOD_FUNCTION` adds `bss_get` to function pointers, but I didn't dig into it.)